CoffeeChat/Controller/SocialMediaViewController.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  SocialMediaViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/5/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class SocialMediaViewController: UIViewController {
+
+    // MARK: - Private Data vars
+    private weak var delegate: OnboardingPageDelegate?
+    private let userDefaults = UserDefaults.standard
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let disclaimerLabel = UILabel()
+//    private var facebookTextField = UITextField()
+//    private var instagramTextField = UITextField()
+    private var facebookTextField: UITextField!
+    private var instagramTextField: UITextField!
```

Comment:
preferably we can avoid force unwrapping;
We can't make the textfield in the `init` though since `getSocialMediaTextField()` needs self to set the textfield change action, but you can make `getSocialMediaTextField` take in the textfield as a parameter instead
That way we can initialize with `UITextField()` then set it up in `viewDidLoad` by calling `getSocialMediaTextField(textview)`